firmware_configurations/klipper/clee/skr-e3-mini-v3.cfg

```diff
@@ -0,0 +1,306 @@
+microsteps: 16
+endstop_pin: ^PC2
+position_endstop: -0.10
+position_max: 120
+position_min: -1.5
+homing_speed: 10
+second_homing_speed: 3.0
+homing_retract_dist: 3.0
+
+[tmc2209 stepper_z]
+uart_pin: PC11
+tx_pin: PC10
+uart_address: 1
+interpolate: False
+## For OMC (StepperOnline) 17LS13-0404E-200G 0.4A 
+#run_current: 0.2
+## For LDO-42STH25-1004CL200E 1.0A
+#run_current: 0.37
+sense_resistor: 0.110
+stealthchop_threshold: 0                                            # Set to 999999 to turn stealthchop on, and 0 to use spreadcycle
+
+#####################################################################
+#   Extruder
+#####################################################################
+
+[extruder]
+step_pin: PB3
+dir_pin: PB4                                                        # Add ! if moving opposite direction
+enable_pin: !PD1
+#full_steps_per_rotation: 200                                       # Set to 200 for LDO 1.8° stepper motor, and set to 400 for OMC(StepperOnline) 0.9° stepper motor
+rotation_distance: 22.23                                            # See calibrating rotation_distance on extruders doc
+gear_ratio: 50:10                                                   # For Mini Afterburner
+microsteps: 64
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC8
+##  Validate the following thermistor type to make sure it is correct
+##  See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options
+#sensor_type: EPCOS 100K B57560G104F                                 # Adjust for your hotend thermistor. See 'sensor types' list at end of file
+sensor_pin: PA0
+control: pid                                                        # Do PID calibration after initial checks
+pid_Kp: 28.182
+pid_Ki: 1.978
+pid_Kd: 100.397
+min_temp: 0
+max_temp: 270
+min_extrude_temp: 170
+max_extrude_only_distance: 150
+max_extrude_cross_section: 0.8
+pressure_advance: 0.0                                               # See tuning pressure advance doc
+pressure_advance_smooth_time: 0.040
+
+[tmc2209 extruder]
+uart_pin: PC11
+tx_pin: PC10
+uart_address: 3
+interpolate: False
+## For OMC (StepperOnline) 14HR07-1004VRN 1A 0.9°
+#run_current: 0.5	# for OMC 14HR07-1004VRN rated at 1A
+## For LDO LDO 36STH17-1004AHG 1A 1.8°
+#run_current: 0.3	# for LDO 36STH17-1004AHG 
+sense_resistor: 0.110
+stealthchop_threshold: 0                                            # Set to 0 for spreadcycle, avoid using stealthchop on extruder
+
+#####################################################################
+#   Bed Heater
+#####################################################################
+
+[heater_bed]
+heater_pin: PC9
+##  Validate the following thermistor type to make sure it is correct
+##  See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options
+#sensor_type: NTC 100K MGB18-104F39050L32                           # For Keenovo, verify yours
```

Comment:
90% of beds are going to end up using `Generic 3950` so it might make sense to set that as the default.